package.json

```diff
@@ -7,9 +7,11 @@
     "ios": "expo start --ios",
     "start": "expo start",
     "test": "jest --verbose --coverage",
-    "web": "expo start --web"
+    "web": "expo start --web",
+    "lint": "tsc --noEmit && eslint --fix --ext .ts,.tsx ./"
```

Comment:
With this script, running `npm run lint` will always fix issues. I don't know if that was your intention.

running `npm run lint --fix` would be the same as running `npm run lint`